src/main/java/org/opentripplanner/transit/raptor/api/request/DebugRequest.java

```diff
@@ -106,6 +110,13 @@ public int debugPathFromStopIndex() {
         return stopArrivalListener;
     }
 
+    /**
+     * Pattern Ride debug event listener.
```

Comment:
Comment should be removed or expanded, it's currently the same as the method name.